int32 LEFT_FRONT = 1
int32 RIGHT_FRONT = 2
int32 LEFT_REAR = 4
int32 RIGHT_REAR = 8
int32 LEFT_MIDDLE = 16
int32 RIGHT_MIDDLE = 32

std_msgs/Header header
int32 selected_legs
geometry_msgs/Vector3 left_front
geometry_msgs/Vector3 right_front
geometry_msgs/Vector3 left_middle
geometry_msgs/Vector3 right_middle
geometry_msgs/Vector3 left_rear
geometry_msgs/Vector3 right_rear
---
bool success
